src/gtnh/modpack_manager.py

```diff
@@ -719,6 +748,38 @@ def set_github_mod_side(self, mod_name: str, side: str) -> bool:
             log.error(f"Release `{Fore.RED}{mod_name} is not a github mod{Fore.RESET}")
             return False
 
+    def set_external_mod_side(self, mod_name: str, side: str) -> bool:
+        if self.assets.has_external_mod(mod_name):
+            mod: ExternalModInfo = self.assets.get_external_mod(mod_name)
+        else:
+            log.error(f"Release `{Fore.RED}{mod_name} is not an external mod{Fore.RESET}")
+            return False
+
+        if mod.side == side:
+            log.warn(f"{Fore.YELLOW}{mod.name}'s side is already set to {side}{Fore.RESET}")
+            return False
+
+        if side in [s.name for s in Side]:
+            mod.side = Side[side]
+            # idk a better way of doing this
```

Comment:
You shouldn't have to do this at all.  The `get_external_mod` should return the actual object, so by updating the mod, you've updated the mod held in the list and the dictionary.

Reply:
ah i see, got too cautious about outdated cache